app/controllers/api/internal/v1/hearing_results_controller.rb

```diff
@@ -5,20 +5,30 @@ module Internal
     module V1
       class HearingResultsController < ApplicationController
         def show
-          @hearing = CommonPlatform::Api::GetHearingResults.call(hearing_id: params[:id])
+          @hearing = CommonPlatform::Api::GetHearingResults.call(
+            hearing_id: permitted_params[:id],
+            sitting_day: permitted_params[:sitting_day],
+          )
+
           render json: HearingSerializer.new(@hearing, serialization_options)
```

Comment:
I know we're deprecating v1, but would it be worth wrapping the render with the conditional guard like we have in v2?

```
if hearing_result.present?
            render json: HmctsCommonPlatform::HearingResulted.new(hearing_result&.body).to_json,
                   status: :ok
          else
            render json: {}, status: :not_found
          end
```

Reply:
VCD are currently not using this endpoint at all. They will only start using the hearing result endpoint when they move to v2.

Reply:
Gotcha. I thought that this was the hearing endpoint renamed. All good